src/store/getters.ts

```diff
@@ -312,8 +317,19 @@ export default {
     return `${name} ${enDash} ${date}`
   },
 
-  /** */
+  /**
+   * Is True if business is in Limited Restoration state, ie
+   * the last filing to change state was a Restoration filing.
+   */
   isEntityInLimitedRestoration (state: StateIF): boolean {
-    return (state.stateFiling?.business?.state === 'ACTIVE' && state.stateFiling.hasOwnProperty('restoration'))
```

Comment:
We don't care about the business state in the state filing (though I think it's the current business state anyway, not the state at the time of the filing).